migrationmanager/migrators/traces/migrations/000025_add_time_column_ttl_top_level.up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE signoz_traces.top_level_operations ON CLUSTER {{.SIGNOZ_CLUSTER}} 
+    ADD COLUMN IF NOT EXISTS time DateTime DEFAULT now() CODEC(ZSTD(1));
+
```

Comment:
are we not considering adding it to the distributed table as well ?

Reply:
I thought about it but I didn't see the need to do it because we just want old ones to get TTL'd on the actual tables but no use of that column in the distributed table. I can add it for consistency.

Reply:
yeah that is fine. don't see an issue